.github/workflows/test.yml

```
@@ -96,7 +96,7 @@ jobs:
       - name: Set up Python 3.13
```

Let's simplify so it doesn't need to be updated manually.

```suggestion
      - name: Set up Python
```